interface/backend/api/views.py

```diff
@@ -42,6 +43,27 @@ class ImageSeriesViewSet(viewsets.ModelViewSet):
     serializer_class = serializers.ImageSeriesSerializer
 
 
+class ImageMetadataApiView(APIView):   
+
+    def get(self, request):
+        '''
+        Get metadata of a DICOM image including the image in base64 format.
+        Example: .../api/images/metadata?dicom_location='FULL_PATH_TO_IMAGE'
+        ---
+        parameters:
+            - name: dicom_location
+            description: 'QUOTED' full location of the image
+            required: true
+            type: string
+        '''
+        path = request.GET.get('dicom_location')
+        if path is None:
+            raise Exception('dicom_location not provided')
```

Comment:
I don't recommend raising ``Exception`` - there is almost always a better one. Indeed, why not just do a raw ``request.GET`` and let it ``KeyError`` and — potentially — reraise it as a ``ValueError``. I mean, that's going to be far more useful to the callsite.